gitgud/levels/intro/__init__.py

```diff
@@ -1,11 +1,11 @@
-from collections import OrderedDict
-
 import pkg_resources
 
 from gitgud.levels.util import BasicChallenge
+from gitgud.levels.util import NamedList
 
-all_challenges = OrderedDict()
-all_challenges['committing'] = BasicChallenge('committing', pkg_resources.resource_filename(__name__, '_committing/'))
-all_challenges['branching'] = BasicChallenge('branching', pkg_resources.resource_filename(__name__, '_branching/'))
-all_challenges['merging'] = BasicChallenge('merging', pkg_resources.resource_filename(__name__, '_merging/'))
-all_challenges['rebasing'] = BasicChallenge('rebasing', pkg_resources.resource_filename(__name__, '_rebasing/'))
+all_challenges = NamedList([
```

Comment:
Make this of type Level, and once again, forego specifying the names in the tuple and instead use the constructor of level to handle the names

Reply:
Rename this to level since that's what the new version will be

Reply:
Resolving this thread becuase I commented again on the new version